src/screens/Onboarding/OnboardingItem.tsx

```diff
@@ -0,0 +1,62 @@
+import {
+  StyleSheet,
+  Text,
+  View,
+  Image,
+  useWindowDimensions,
+  ImageSourcePropType,
+} from "react-native";
+import { COLORS, FONTS, SIZES } from "../../styles/theme";
+
+interface Props {
+  id: string;
+  title: string;
+  description: string;
+  image: ImageSourcePropType;
+}
+
+const OnboardingItem: React.FC<Props> = ({ id, title, description, image }) => {
+  const { width } = useWindowDimensions();
+  return (
+    <View style={(styles.container, { width })}>
+      <Image
+        source={image}
+        style={[styles.image, { width, resizeMode: "contain" }]}
+      />
+      <View style={{ flex: 0.3 }}>
+        <Text style={styles.title}>{title}</Text>
+        <Text style={styles.description}>{description}</Text>
+      </View>
+    </View>
+  );
+};
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    backgroundColor: "#fff",
+    alignItems: "center",
+    justifyContent: "center",
+  },
+  image: {
+    flex: 0.7,
+    justifyContent: "center",
+  },
+  title: {
+    color: COLORS.primary,
+    fontSize: 30,
+    fontWeight: "bold",
```

Comment:
@Alphonce-Mutebi @Chegemartinx i wonder why we specify font size and weight here.

isn't that information for a component that we will import when building this page? 

same question for the "description" object below.

Also, adding a few comments here and there can be helpful.
first for allen (and me). but build with future team members in mind. when they join, they should easily find their way around.

and: it is an open source project in the end. so basically everybody (maybe even auditors) shoudld easily find their way around.

Reply:
oh, i guess you are building the components for all intro pages here.
but the question about styling still stands.

and when it is a component, I guess it should not be in the screens folder.

the screens folder should only contain the actual screens. because everything is could be reused.

example: When you look at the designs, the "governance" section is actually very similar to the intro swipe screens.
so you need to have these in a centralized folder, and then import them into both, the onboarding intro screens and the governance screens
@Alphonce-Mutebi

Reply:
Noted